src/lib/blueprint.ts

```diff
@@ -127,18 +135,37 @@ interface ObjectProperty extends BaseProperty {
 
 type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
 
-export interface TypesModule {
-  openapi: Openapi
+interface Context {
+  codeSamples: CodeSample[]
 }
 
-export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
+export const TypesModuleSchema = z.object({
+  codeSampleDefinitions: z.array(CodeSampleDefinitionSchema).default([]),
```

Comment:
I don’t remember discussing this in our meeting, but am I correct in understanding that code sample definitions will come from the types module? Will they be populated manually?

Reply:
@andrii-balitskyi Isn't this the answer? https://hello-seam.slack.com/archives/C0625RH4MMM/p1721690567582069?thread_ts=1721690459.039339&cid=C0625RH4MMM

Reply:
Oh I missed that text. Thank you!